### PR TITLE
adding navigation buttons - closes #17

### LINKED ISSUE #17: add buttons to GH pages
@gvn: can you add button on each GH page that sends you back to the Foundation site, and add
button for the next activity at the bottom of each activity?

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
@ScottDowne can you review this?

File: _layouts/default.html [hunk 1]
```diff
@@ -42,6 +42,29 @@ <h2>Web Literacy</h2>
           </div>
           <div id="content" class="col-md-9 col-xs-12">
             {{ content }}
+
+            {% if page.previous.url and page.next.url %}
```
Comment (ScottDowne) [inline _layouts/default.html]: Shouldn't this be an "or" and not an "and"?

File: css/main.css [hunk 2]
```diff
@@ -800,3 +800,21 @@ video {
...
+.next-prev {
+  display: flex;
+  background: #f2f2f2;
+  padding: 20px;
+  justify-content: space-between;
+  text-transform: uppercase;
+  margin: 30px 0;
+  color: #8f8f8f;
+}
+  .next-prev .wrapper {
```
Comment (ScottDowne) [inline css/main.css]: Just some whitespace that could be cleaned up it looks like.

**Review by ScottDowne — CHANGES_REQUESTED** [review]
Left two comments.

**Review by ScottDowne — APPROVED** [review]
Looks good!

**Pull request merged**